### PR TITLE
Add initial docs for utilization reports

### PR DESCRIPTION
**Pull request by Jkrzy**
## Description

For #981, initial pass at a plain language description of how Tock is currently reporting on utilization.

### CODE REVIEW

File: docs/utilization.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Utilization reports in Tock
+
+[:arrow_left: Back to Tock Documentation](../docs)
+
+
+Tock utilization reports display progress towards billing targets
+for each user, business unit, and the overall organization.
+
+Each user has a billing target which is the percentage of worked hours that are expected to be tocked to billable projects.
```
Comment (tbaxter-18f) [inline docs/utilization.md:9]: Can't remember: in the admin, do we express the target as a number of hours, or as a percentage?

File: docs/utilization.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+### BILLING EXPECTATION
+
+The percent of `TOTAL HOURS` which are expected to be billable for a given reporting period. Set per user, per timecard, and will vary as an individual's role, position, or organization change.
+
+**Default: 80%**
+
+## Example
+
+Stacey is an active and billable 18F employee, here's a look at their recent tock activity! As an independent contributor, Stacey has a `BILLING_EXPECTATION` of 80%.
```
Comment (tbaxter-18f) [inline docs/utilization.md:47]: Minor, but I think you may have meant "individual contributor"

File: docs/utilization.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# 40 * 0.80 = 25.6
+
+utilization = billable_hours / target_billable_hours
+# 28 / 25.6 = 1.09
+```
+
+### Weeks 1-4
+Stacey's `utilization` over the last 4 weeks is 95%. Stacey tocked 116 of an expected ~122 billable hours over the 4 week period.
+
+Stacey had 8 hours of `Out of Office` time in week 2 which is excluded from all totals, notice the `total_hours` is 152 for the 4 weeks.
```
Comment (tbaxter-18f) [inline docs/utilization.md:100]:
```suggestion
Stacey had eight hours of `Out of Office` time in week 2, which is excluded from all totals. Notice the `total_hours` is 152 for the 4 weeks.
```

File: docs/utilization.md (hunk 1 above)
Comment (Jkrzy) [reply docs/utilization.md:9]: Percentage

File: docs/utilization.md (hunk 2 above)
Comment (Jkrzy) [reply docs/utilization.md:47]: Ah yes!

**Comment by Jkrzy** [comment]
Thanks @tbaxter-18f, made the recommended changes!

**Pull request merged**